sydent/db/threepid_associations.sql

```diff
@@ -40,7 +39,6 @@ CREATE TABLE IF NOT EXISTS global_threepid_associations (
     originId integer not null,
     sgAssoc text not null
 );
-CREATE INDEX IF NOT EXISTS lookup_hash_medium on global_threepid_associations (lookup_hash, medium);
 CREATE INDEX IF NOT EXISTS medium_address on global_threepid_associations (medium, address);
 CREATE INDEX IF NOT EXISTS medium_lower_address on global_threepid_associations (medium, lower(address));
```

Comment:
The migration code has an `IF EXISTS` so we may as well delete `medium_address` and `medium_lower_address ` here I think, since they're entirely wrong and just liable to confuse anyone else who adds indicies later. wdyt?

Reply:
Arguably should also delete the other lines to add the `lookup_hash_medium` indicies in the migration code for the same reason, if you add an `IF EXISTS` (github won't let me comment on lines you haven't changed...)